python/tk_multi_reviewsubmission/renderer.py

```diff
@@ -54,7 +59,7 @@ def gather_nuke_render_info(self, path, output_path,
                                 width, height,
                                 first_frame, last_frame,
                                 version, name,
-                                color_space):
+                                color_space, tmp_nuke_file_path):
```

Comment:
Since this is technically a public api method, how about just calling this `burnin_nk`?

Reply:
Done.